public/API/API_GetTicketData.php

```diff
@@ -0,0 +1,131 @@
+<?php
+require_once __DIR__ . '/../../vendor/autoload.php';
+
+runPublicApiMiddleware();
+
+$baseUri = '/ticketmanager.php?';
```

Comment:
We could have the full URL here by prepending the application's URL. There's no need to build the full URL in any client this way.
`$baseUri = getenv('APP_URL') . '/ticketmanager.php?';`